synapseclient/client.py

```diff
@@ -2049,6 +2068,7 @@ def submit(self, evaluation, entity, name=None, team=None, silent=False, submitt
                                 In the absent of this parameter, the entity name will be used.
         :param team:            (optional) A :py:class:`Team` object, ID or name of a Team that is registered for the
                                 challenge
+        :param dockerTag:       (optional) The Docker tag must be specified if the entity is a DockerRepository. Defaults to "latest".
```

Comment:
Argument description should suggest how to use the argument. Having both "optional" and "must be specified" can be confusing. Maybe "(optional) Set the docker tag to use in submitting a DockerRepository. Default to 'latest'."  In this case, it's optional because the code taking care of the case when the user doesn't specify a value. So "must be specified" is not true here.